lib/grinder_sdk.dart

```diff
@@ -44,7 +44,7 @@ class Dart {
   /// Returns the stdout.
   static String run(String script,
       {List<String> arguments : const [], bool quiet: false,
-       String packageRoot, String workingDirectory, int vmNewGenHeapMB,
```

Comment:
Do we want to keep `workingDirectory` as a deprecated parameter for now, and use it when provided, and throw if both `workingDirectory` and `runOptions` are non-null?